src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -106,14 +111,25 @@ function Test-SsisAzure-IntegrationRuntime
         $description = "SSIS-Azure integration runtime"
 
         # Replace the following three variables to real values in record mode
-        $catalogServerEndpoint = 'fakeserver'
-        $catalogAdminUsername = 'fakeuser'
-        $catalogAdminPassword = 'fakepassord'
+        $catalogServerEndpoint = $Env:CatalogServerEndpoint
```

Comment:
Why are these now environment variables? We want to avoid environment variables as much as possible as it makes batch rerecording very complicated.  Instead, perhaps you should just add an element of randomness to the names (using getAssetName)

Reply:
The credentials here should point to real SQL to be able to record. @zhangyd2015

Reply:
Yes, when recording we need a real Azure SQL database to be configured here.

Reply:
The flow of creating a new azure SQL database in test is out of our scenario. This is always required, somehow got deleted in the middle, now bring it back.

On another hand, recording is a low frequency operation, we intend to prepare such resources as prerequisite.

Reply:
@maddieclayton  Where do we add these instructions?